modules/phase_field/tests/KKS_system/kks_multiphase.i

```diff
@@ -291,6 +298,13 @@
     args      = 'eta1 eta3'
     mob_name  = 1
   [../]
+  [./mult_CoupledACint_2]
+    type = SimpleCoupledACInterface
```

Comment:
Yeah, that makes sense.

On Tue, Jul 19, 2016, 4:08 PM laagesen <notifications@github.com> wrote:

> In modules/phase_field/tests/KKS_system/kks_multiphase.i
> <https://github.com/idaholab/moose/pull/7365#discussion_r71433487>:
>
> > @@ -291,6 +298,13 @@
> >      args      = 'eta1 eta3'
> >      mob_name  = 1
> >    [../]
> > +  [./mult_CoupledACint_2]
> > +    type = SimpleCoupledACInterface
>
> Well, a lot of what's in this test is specific to the KKS multi-phase
> system. So I think it makes more sense for the kks_multiphase test to stay
> with the other KKS_system tests. However I could also add a separate,
> simpler test that's more specific to SimpleCoupledACInterface and put it
> with SimpleACInterface, what do you think?
>
> —
> You are receiving this because you were assigned.
>
>
> Reply to this email directly, view it on GitHub
> <https://github.com/idaholab/moose/pull/7365/files/0cf74967289857644be53d2d9a14579999333871#r71433487>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAMWPuRUvEfVda3xdZ_t7lPXqmlpxZOuks5qXUrFgaJpZM4JQI8A>
> .
>